app/client/components/wizardRouterAdapter.tsx

```diff
@@ -64,22 +75,23 @@ export const ReturnToYourProductButton = (
 const RootComponent = (props: RootComponentProps) => (
   <>
     <PageContainer>
-      {props.routeableStepProps.stepLabels ? (
-        <ProgressBreadcrumb
-          current={props.routeableStepProps.currentStep}
-          labels={props.routeableStepProps.stepLabels}
-        />
-      ) : (
-        <PageContainerSection>
-          <ProgressCounter
+      {!!props.routeableStepProps.currentStep &&
```

Comment:
`!!`? Is this not not or am I just unaware of something?

Reply:
Correct, it is a NOT NOT, and is used to force a truthy (i.e. null/undefined or actual object) into a boolean.

In this use case it forms the opening part of some [React Conditional Rendering](https://reactjs.org/docs/conditional-rendering.html)